boards.txt

```diff
@@ -50155,3 +50155,157 @@ cyobot_v2_esp32s3.menu.ZigbeeMode.zczr.build.zigbee_mode=-DZIGBEE_MODE_ZCZR
+sat1core_rev5_1.menu.FlashMode.qio.build.boot=qio
+sat1core_rev5_1.menu.FlashMode.qio.build.boot_freq=80m
+sat1core_rev5_1.menu.FlashMode.qio.build.flash_freq=80m
+
+sat1core_rev5_1.menu.FlashSize.16M=16MB (128Mb)
+sat1core_rev5_1.menu.FlashSize.16M.build.flash_size=16MB
+
+sat1core_rev5_1.menu.LoopCore.1=Core 1
+sat1core_rev5_1.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+sat1core_rev5_1.menu.LoopCore.0=Core 0
+sat1core_rev5_1.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+sat1core_rev5_1.menu.EventsCore.1=Core 1
+sat1core_rev5_1.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+sat1core_rev5_1.menu.EventsCore.0=Core 0
+sat1core_rev5_1.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+sat1core_rev5_1.menu.USBMode.default=Hardware CDC and JTAG
+sat1core_rev5_1.menu.USBMode.default.build.usb_mode=1
+sat1core_rev5_1.menu.USBMode.hwcdc=USB-OTG (TinyUSB)
+sat1core_rev5_1.menu.USBMode.hwcdc.build.usb_mode=0
+
+sat1core_rev5_1.menu.CDCOnBoot.cdc=Enabled
+sat1core_rev5_1.menu.CDCOnBoot.cdc.build.cdc_on_boot=1
+sat1core_rev5_1.menu.CDCOnBoot.default=Disabled
+sat1core_rev5_1.menu.CDCOnBoot.default.build.cdc_on_boot=0
+
+sat1core_rev5_1.menu.MSCOnBoot.default=Disabled
+sat1core_rev5_1.menu.MSCOnBoot.default.build.msc_on_boot=0
+sat1core_rev5_1.menu.MSCOnBoot.msc=Enabled (Requires USB-OTG Mode)
+sat1core_rev5_1.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+sat1core_rev5_1.menu.DFUOnBoot.default=Disabled
+sat1core_rev5_1.menu.DFUOnBoot.default.build.dfu_on_boot=0
+sat1core_rev5_1.menu.DFUOnBoot.dfu=Enabled (Requires USB-OTG Mode)
+sat1core_rev5_1.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+sat1core_rev5_1.menu.UploadMode.default=UART0 / Hardware CDC
+sat1core_rev5_1.menu.UploadMode.default.upload.use_1200bps_touch=false
+sat1core_rev5_1.menu.UploadMode.default.upload.wait_for_upload_port=false
+sat1core_rev5_1.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+sat1core_rev5_1.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+sat1core_rev5_1.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme1=Small App w/ OTA + Huge FS (2MB APP/2MB OTA/12MB SPIFFS)
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme1.build.custom_partitions=gen4esp32_2MBapp_2MBota_12MBspiffs
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme1.upload.maximum_size=2097152
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme2=Medium App w/ OTA + Large FS (4MB APP/4MB OTA/7MB SPIFFS)
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme2.build.custom_partitions=gen4esp32_4MBapp_4MBota_7MBspiffs
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme2.upload.maximum_size=4718592
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme3=Large App w/ OTA (8MB APP/8MB OTA)
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme3.build.custom_partitions=gen4esp32_8MBapp_8MBota
+sat1core_rev5_1.menu.PartitionScheme.gen4esp32scheme3.upload.maximum_size=8323072
```

Comment:
This won't work, as those partitions does not exist in the core. You have to use the ones we have and provide a "Custom" option there also, where users can you your custom ones you can provide in your docs or some repo :)